README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the projects Aston Yong will work on throughout SPS.
```

Comment:
nit: This PR does a few more things than just add a ReadMe. It should probably be named something like "Implements basic portfolio page" or similar.